src/main/java/io/kf/coordinator/model/AuthorizedTaskRequest.java

```diff
@@ -5,14 +5,16 @@
 import lombok.NonNull;
 import lombok.Value;
 
+import java.util.List;
+
 @Value
 @Builder
 public class AuthorizedTaskRequest {
-  @NonNull private TaskAction action;
-
-  @NonNull private String task_id;
-
-  private String release_id;
-
-  private String accessToken;
+    @NonNull
+    TaskAction action;
```

Comment:
why did you removed private?

Reply:
lombok annotations seem to take care of this for us